packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -541,12 +565,36 @@ export class StatsAnalyzer extends EventsScope {
               `StatsAnalyzer:index#compareLastStatsResult --> No audio Energy present`,
               currentStats.totalAudioEnergy
             );
+            if (this.lastEmittedStartStopEvent.audio.remote === EVENTS.LOCAL_MEDIA_STARTED) {
+              this.emit(
+                {
+                  file: 'statsAnalyzer',
+                  function: 'compareLastStatsResult',
+                },
+                EVENTS.NO_AUDIO_SENT,
+                {
+                  mediaType: 'audio',
+                }
+              );
+            }
           }
 
           if (currentStats.audioLevel === 0) {
             LoggerProxy.logger.info(
               `StatsAnalyzer:index#compareLastStatsResult --> audio level is 0 for the user`
             );
+            if (this.lastEmittedStartStopEvent.audio.remote === EVENTS.REMOTE_MEDIA_STARTED) {
```

Comment:
this doesn't look right - why are we checking for remote media here? is `currentStats.audioLevel` for the audio level of the local user's audio or for the incoming remote audio? I thought it was for local audio and also the event that we're sending in line 587 is `NO_AUDIO_SENT`, so what does this have to do with remove audio?

Reply:
Yes, this should check for local event not remote. Fixed this.